templates/docs/utilities/text-max-width.md

```diff
@@ -0,0 +1,36 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Text max width | Utilities
+---
+
+# Text max width
+
+<hr>
+
+To limit the width of text elements, such as lists to match the max width of paragraphs and headings use the `.u-text-max-width` utility class on the list element itself, or on a parent element.
```

Comment:
```suggestion
We aim to limit text to a measure (line width) of around 90 characters. This is done for readability purpouses, and is already applied to paragraphs and headings.

There are cases where we place text inside content-agnostic elements like lists for example. While we can't put a blanket `max-width`on those elements, because this would affect lists of elements other than text, we do need to ensure text placed in those elements does not exceed the recommended measure.

To do  so, use the `.u-text-max-width` utility class to limit the measure of any element that contains text.
```

Reply:
```suggestion
We aim to limit text to a measure (line width) of around 90 characters. This is done for readability purposes, and is already applied to paragraphs and headings.

To limit the width of text lists to match the line width of paragraphs use the `.u-text-max-width` utility class on the list element itself, or on a parent element.
```